src/topology/compact_open.lean

```diff
@@ -88,6 +88,22 @@ lemma continuous_comp : continuous (continuous_map.comp g : C(α, β) → C(α,
 continuous_generated_from $ assume m ⟨s, hs, u, hu, hm⟩,
   by rw [hm, preimage_gen g hs hu]; exact continuous_map.is_open_gen hs (hu.preimage g.2)
 
+variable (f : C(α, β))
+
+private lemma image_gen {s : set α} (hs : is_compact s) {u : set γ} (hu : is_open u) :
+  (λ (g : C(β, γ)), g.comp f) ⁻¹' (compact_open.gen s u) =
+    compact_open.gen (f '' s) u :=
```

Comment:
Reiteratnig my comment above; may as well put this on one line, and drop the unecessary parens
```suggestion
  (λ g : C(β, γ), g.comp f) ⁻¹' compact_open.gen s u = compact_open.gen (f '' s) u :=
```